front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -195,6 +203,17 @@ class PipelineController {
     }
 
     checkForDuplicatePipeline(pipeline.getApplication(), pipeline.getName().trim(), pipeline.getId())
+
+    def errors = new GenericValidationErrors(pipeline)
+    pipelineValidators.each {
+      it.validate(pipeline, errors)
+    }
+
+    if (errors.hasErrors()) {
+      throw new ValidationException(
```

Comment:
There's built-in support already for `ValidationException` in `kork`.

Applications are doing validations with an Application-specific exception, opt'd not to follow that pattern. We can clean that usage up subsequently as well.

Reply:
```
{
    "timestamp": 1552003316289,
    "status": 400,
    "error": "Bad Request",
    "exception": "com.netflix.spinnaker.kork.web.exceptions.ValidationException",
    "message": "No message available",
    "errors": ["your error message here"]
}
```

Serialized response fits the pattern we've got elsewhere.